index.js

```diff
@@ -1,24 +1,29 @@
 'use strict';
 
-var path = require('path');
-var isglob = require('is-glob');
-var pathDirname = require('path-dirname');
+var dirname = require('path').posix.dirname;
```

Comment:
I say we use less dependencies. Why is this needed? Nodejs v6 is getting its EOL in 2 weeks and you're talking about v0.10?

Reply:
Because node 0.10 was never properly LTS'd and gulp 4 was in development long before the new release phases were announced.  Also people still use it: https://github.com/rvagg/through2/issues/95

Don't purposefully break your users 😦

Reply:
Users with ancient node versions can use current glob-parent version. What's the issue here?

I'm pushing chokidar v3 to tens of millions of installs with `fsevents v2` — which requires `napi 4` - which is only present in node v8.16+. Node 8.16 would be the **lowest** supported node version since apr 2019. It radically simplifies installation times / gyp builds, APIs and all sorts of things. Anyone who doesn't like that can stay on v2.

**Any additional dependency is a security threat!** Which is why i've been always striving to reduce the amount of deps.

Reply:
Did you miss the part where I said I'd cut a release with your changes but including the path-dirname module THEN cut a release without it and an updated build matrix?

Your aggressiveness is not appreciated...

Reply:
Yeah, I missed that. Sorry for that. Let's release the minor with path-dirname and the major without it.

Reply:
@paulmillr due to #18, I'll have to release as a major and then another major, but that should be fine too.